src/main/java/org/jfrog/hudson/trigger/ArtifactoryMultibranchTrigger.java

```diff
@@ -76,7 +76,12 @@ public DescriptorImpl getDescriptor() {
 
         @Override
         public boolean isApplicable(Item item) {
-            return item instanceof MultiBranchProject;
+            try {
+                return item instanceof MultiBranchProject;
+            } catch (NoClassDefFoundError ignore) {
```

Comment:
Notice that the ```MultiBranchProject``` class is also referenced as an import of this class and also in the super-class this class extends. Are you sure this also won't be a cause for a  ```NoClassDefFoundError``` as well?
The same goes for the ```ArtifactoryTrigger``` class as well.

In case this indeed causes an issue, you can look at what we did for the *jenkins-multijob-plugin* - 

```
    @Override
    public boolean isApplicable(AbstractProject<?, ?> item) {
        this.item = item;
        Class<?> itemClass = item.getClass();
        return itemClass.isAssignableFrom(FreeStyleProject.class) || itemClass.isAssignableFrom(MatrixProject.class) ||
                (Jenkins.get().getPlugin(PluginsUtils.MULTIJOB_PLUGIN_ID) != null &&
                        itemClass.isAssignableFrom(MultiJobProject.class)) ||
                PluginsUtils.PROMOTION_BUILD_PLUGIN_CLASS.equals(itemClass.getSimpleName());
    }
```

Reply:
It doesn't look like an issue in this case. I tested it on my environment:
![image](https://user-images.githubusercontent.com/11367982/126062116-598eed06-e82d-4e04-b70d-0a719523a8c1.png)

Besides, the error message was thrown on ArtifactoryTrigger.java:48 and not on the import.